apps/tests/devapps/managedidentity_sample.go

```diff
@@ -3,36 +3,18 @@ package main
 import (
 	"context"
 	"fmt"
-	"net/http"
 
 	mi "github.com/AzureAD/microsoft-authentication-library-for-go/apps/managedidentity"
 )
 
 func RunManagedIdentity() {
-	customHttpClient := &http.Client{}
-
-	miSystemAssigned, error := mi.New(mi.SystemAssigned())
-	if error != nil {
-		fmt.Println(error)
-	}
-
-	miClientIdAssigned, error := mi.New(mi.ClientID("client id 123"), mi.WithHTTPClient(customHttpClient))
-	if error != nil {
-		fmt.Println(error)
+	miSystemAssigned, err := mi.New(mi.SystemAssigned())
+	if err != nil {
+		fmt.Println(err)
 	}
-
-	miResourceIdAssigned, error := mi.New(mi.ResourceID("resource id 123"))
-	if error != nil {
-		fmt.Println(error)
+	temp, err := miSystemAssigned.AcquireToken(context.Background(), "https://management.azure.com/")
+	if err != nil {
+		println(err.Error())
 	}
-
-	miObjectIdAssigned, error := mi.New(mi.ObjectID("object id 123"))
-	if error != nil {
-		fmt.Println(error)
-	}
-
-	miSystemAssigned.AcquireToken(context.Background(), "resource", mi.WithClaims("claim"))
-	miClientIdAssigned.AcquireToken(context.Background(), "resource")
-	miResourceIdAssigned.AcquireToken(context.Background(), "resource", mi.WithClaims("claim"))
-	miObjectIdAssigned.AcquireToken(context.Background(), "resource")
+	fmt.Println("token : ", temp.AccessToken)
```

Comment:
Pls do not "print" access tokens. It's bad practice. Better to print the token expiration time instead and maybe the token source (cache / identity provider).